math/mathcore/src/TMath.cxx

```diff
@@ -3151,19 +3154,788 @@ Double_t TMath::VavilovDenEval(Double_t rlam, Double_t *AC, Double_t *HC, Int_t
    return v;
 }
 
-
 //explicitly instantiate template functions from VecCore
-#ifdef R__HAS_VECCORE
-#include <Math/Types.h>
+//#ifdef R__HAS_VECCORE
 template ROOT::Double_v vecCore::math::Sin(const ROOT::Double_v & x);
 template ROOT::Double_v vecCore::math::Cos(const ROOT::Double_v & x);
+template ROOT::Double_v vecCore::math::Tan(const ROOT::Double_v & x);
 template ROOT::Double_v vecCore::math::ASin(const ROOT::Double_v & x);
 template ROOT::Double_v vecCore::math::ATan(const ROOT::Double_v & x);
+
+template ROOT::Double_v vecCore::math::Exp(const ROOT::Double_v & x);
+
 template ROOT::Double_v vecCore::math::ATan2(const ROOT::Double_v & x,const ROOT::Double_v & y);
+
 // missing in veccore
-// template ROOT::Double_v vecCore::math::ACos(const ROOT::Double_v & x);
 // template ROOT::Double_v vecCore::math::Sinh(const ROOT::Double_v & x);
 // template ROOT::Double_v vecCore::math::Cosh(const ROOT::Double_v & x);
 // template ROOT::Double_v vecCore::math::Tanh(const ROOT::Double_v & x);
-// template ROOT::Double_v vecCore::math::Cbrt(const ROOT::Double_v & x);
-#endif
+namespace TMath {
+////////////////////////////////////////////////////////////////////////////////
+::ROOT::Double_v Log2(::ROOT::Double_v &x)
+{
+   return vecCore::math::Log(x)/2.0f;
```

Comment:
This implementation is wrong. The `Log2()` function (equivalent to [`std::log2`](https://en.cppreference.com/w/cpp/numeric/math/log2)) is log base 2, not log divided by 2.

Reply:
No, it should be `vecCore::math::Log2(x)`, which was already present in the 0.4.2 release, but has been improved in master branch of VecCore. That's why we started working there, to enable those functions to be usable here.
https://github.com/root-project/veccore/blob/v0.4.2/include/VecCore/VecMath.h#L314